dev/tasks/r/github.linux.cran.yml

```diff
@@ -33,6 +33,8 @@ jobs:
           - debian-gcc-release
           - fedora-gcc-devel
           - fedora-clang-devel
+          - clang17
+          - clang18
```

Comment:
These are the new rhub container approach https://github.com/r-hub/containers we might be able to change most of our other rhub images to these.

It's nice that they are built daily so we don't have to build them ourselves.